.prettierrc.json

```diff
@@ -5,7 +5,6 @@
   "semi": true,
   "jsxSingleQuote": false,
   "arrowParens": "always",
-  "tailwindFunctions": [
-    "classNames"
-  ]
-}
\ No newline at end of file
+  "tailwindFunctions": ["classNames"],
+  "plugins": ["prettier-plugin-tailwindcss"]
```

Comment:
We removed this from here and put it in lint-staged so that it does not annoy while developing

Reply:
I've intentionally re-enabled it. It's enforcing the official recommended order. It's good to keep that, to better define the classNames.

I think you can override in your VS Code locally.